README.md

```diff
@@ -91,13 +91,13 @@ https://decred.org/downloads/
   to download, build, and install without needing to clone the repo. Run:
 
   ```sh
-  $ go install decred.org/dcrwallet/v4@master
+  $ go install decred.org/dcrwallet/v5@master
   ```
 
   to build the latest master branch, or:
 
   ```sh
-  $ go install decred.org/dcrwallet/v3@latest
+  $ go install decred.org/dcrwallet/v5@latest
```

Comment:
```suggestion
  $ go install decred.org/dcrwallet/v4@latest
```

Reply:
correct, i didn't read it closely, assumed it was missed in the previous bump.